README.md

```diff
@@ -84,14 +87,31 @@ return [
 
 ## Usage
 
-Add the `HasStatuses` trait to a model you like to use statuses on.
+Create The Enum that holds all of your available statuses for the model. \
+The enum must be string backed
+
+```php
+enum ModelStatusEnum: string
+{
+    case PENDING = "pending";
```

Comment:
Could you use PascalCase for the enum cases everywhere?

Reply:
Absolutely!